test-suite/inflation.cpp

```diff
@@ -971,23 +991,103 @@ void InflationTest::testPeriod() {
 
     Date d;
     Frequency f;
-    /* std::pair<Date,Date> res; */
+    std::pair<Date,Date> res;
+
+    // random year between 1950 and 2050
+    srand (time(NULL));
+    int year = rand() %100 + 1950;
```

Comment:
The test looks fast enough.  We could loop from 1950 to 2050 instead of taking a random year.